ONSdigital/design-system
# Replace highlight with prismjs

### What is the context of this PR?
Due to the security issue in highlightjs v9. I investigated into upgrading to v10 but v10 doesnt support ie11. So this PR is to replace highlightjs with prismjs which does support ie11.

As part of this I needed to rename the `tag` css class used in the find a support centre pattern

### How to review 
- Check that code blocks in example tabs look correct and code is styled
- Check that find a support centre pattern still looks as it should
